libeos-payg/manager-service.c

```diff
@@ -999,12 +999,19 @@ epg_manager_service_manager_add_code (EpgManagerService     *self,
   const gchar *code_str;
   g_variant_get (parameters, "(&s)", &code_str);
 
+  g_message ("Trying to enter code %s", code_str);
   epg_provider_add_code (self->provider, code_str, &time_added, &local_error);
 
   if (local_error != NULL)
-    g_dbus_method_invocation_return_gerror (invocation, local_error);
+    {
+      g_warning ("Failed to enter code %s: %s", code_str, local_error->message);
```

Comment:
I still don't think this is warning-worthy; I tend to think of a warning as a "soft assertion failure" rather than "a string was passed to a D-Bus method that I didn't like". GLib has a runtime flag to make warnings fatal to aid debugging, [`G_DEBUG=fatal-warnings`](https://docs.gtk.org/glib/running.html) and some projects like to set this on their test suite; having a warning here would prevent that.

Reply:
(I should have been clearer in my original comment.)

Reply:
Oh also, typo in commit message: `s/attemps/attempts/`

Reply:
Addressed both!